unicorecmsmama/static/css/style.css

```diff
@@ -27,6 +28,7 @@
 /*  NAV  */
 
 	#nav { padding: 10px 16px; background: #f7a11a; font-size: 14px; font-weight: bold; color: #ffc20f; text-align: left; }
+	[dir='rtl'] #nav { padding: 10px 16px; background: #f7a11a; font-size: 14px; font-weight: bold; color: #ffc20f; text-align: right; }
```

Comment:
See above (alternative phrasing: "Use [the Cascade](https://developer.mozilla.org/en-US/docs/Web/CSS/Cascade), Nate.". This could just be:

> [dir='rtl'] #nav { text-align: right; }